### PR TITLE
10203: Remove prod flag and set feature flag to ON for legislative form updates #25701

### PR DESCRIPTION
**Pull request by zurbergram**
## Description

https://github.com/department-of-veterans-affairs/va.gov-team/issues/25701

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/edu-benefits/components/EducationWizard.jsx [hunk 1]
```diff
@@ -390,22 +390,22 @@ export default class EducationWizard extends React.Component {
...
-                          pursuing a teaching certification.{' '}
+                          working toward a teaching certification, <b>or</b>
+                        </li>{' '}
+                        <li className="li-styling">
+                          {' '}
+                          You've already earned a STEM bachelor's or graduate
+                          degree and are pursuing a covered clinical training
+                          program for health care professionals. <br />
                           <a
+                            aria-label="See eligible degree programs, opening in new tab"
```
Comment (shawkey-daniel) [inline src/applications/edu-benefits/components/EducationWizard.jsx:403]: Shouldn't the aria-label include "and clinical training"?

File: src/applications/edu-benefits/components/EducationWizard.jsx [hunk 2]
```diff
@@ -390,22 +390,22 @@ export default class EducationWizard extends React.Component {
...
                             href="https://benefits.va.gov/gibill/docs/fgib/STEM_Program_List.pdf"
                             rel="noopener noreferrer"
                             target="_blank"
-                            onClick={() =>
-                              recordEvent({
-                                event: 'edu-navigation',
-                                'edu-action': 'see-approved-stem-programs',
-                              })
-                            }
```
Comment (shawkey-daniel) [inline src/applications/edu-benefits/components/EducationWizard.jsx:406]: Why is the recording of the event being removed? Did I miss something?

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: src/applications/edu-benefits/components/EducationWizard.jsx (hunk 2 above)
Comment (zurbergram) [reply src/applications/edu-benefits/components/EducationWizard.jsx:406]: I pulled the version of this from https://github.com/department-of-veterans-affairs/vets-website/pull/17642 and then removed Prod Flag

Comment (zurbergram) [reply src/applications/edu-benefits/components/EducationWizard.jsx:406]: Any issues with new content are present in that PR

Comment (ksantiagoBAH) [reply src/applications/edu-benefits/components/EducationWizard.jsx:406]: I went ahead and updated my PR https://github.com/department-of-veterans-affairs/vets-website/pull/17642 to have the onclick record event

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: src/applications/edu-benefits/components/EducationWizard.jsx (hunk 1 above)
Comment (zurbergram) [reply src/applications/edu-benefits/components/EducationWizard.jsx]: Fixed merged in from master

**Review by ksantiagoBAH — APPROVED** [review]

**Pull request merged**